cli/tools/test.rs

```diff
@@ -1361,8 +1361,7 @@ pub async fn run_tests_with_watch(
     let paths_to_watch_clone = paths_to_watch.clone();
     let files_changed = changed.is_some();
     let test_options = &test_options;
-
-    let ps = ps.borrow().clone();
+    let ps = ps.borrow();
```

Comment:
Could you add back this clone? Cloning the ProcState is super fast because it is in an `Arc` and this could cause a BorrowMut error if the code changes.

Reply:
Thanks for pointing this out, I reverted the changes (also in `bench.rs`).